### PR TITLE
Rename member "uuid" to "member_id"

### PR DESCRIPTION
**Pull request by malakai97**

### CODE REVIEW

File: dist/swagger.yaml [hunk 1]
```diff
@@ -1018,10 +1018,10 @@ paths:
         '401': {$ref: '#/responses/401'}
         '403': {$ref: '#/responses/403'}
         '404': {$ref: '#/responses/404'}
-  /member/{uuid}/bags:
+  /member/{member_id}/bags:
```
Comment (dazza-codes) [inline dist/swagger.yaml:1021]: `/member/{id}/bags` ?

File: dist/swagger.yaml [hunk 2]
```diff
@@ -983,10 +983,10 @@ paths:
         '401': {$ref: '#/responses/401'}
         '403': {$ref: '#/responses/403'}
         '409': {$ref: '#/responses/409'}
-  /member/{uuid}:
+  /member/{member_id}:
```
Comment (dazza-codes) [inline dist/swagger.yaml:986]: `/member/{id}` ?

File: dist/swagger.yaml [hunk 3]
```diff
@@ -714,7 +714,7 @@ parameters:
   member_uuid: {name: member, in: query, description: Filter by member_id, required: false,
     type: string, format: uuid}
   node_uuid: {$ref: node_uuid.yml}
-  member_uuid_path: {name: member_uuid, in: path, description: member_id of the member,
+  member_uuid_path: {name: member_id, in: path, description: member_id of the member,
```
Comment (dazza-codes) [inline dist/swagger.yaml:717]: `member_id_path` ?

File: dist/swagger.yaml [hunk 4]
```diff
@@ -1018,10 +1018,10 @@ paths:
         '401': {$ref: '#/responses/401'}
         '403': {$ref: '#/responses/403'}
         '404': {$ref: '#/responses/404'}
-  /member/{uuid}/bags:
+  /member/{member_id}/bags:
     get:
       summary: Returns the member's bags
-      description: 'Returns the member''s bags.  Behavior should match that of ```/bags/?member=<member:uuid>```,
+      description: 'Returns the member''s bags.  Behavior should match that of ```/bags/?member=<member:member_id>```,
```
Comment (dazza-codes) [inline dist/swagger.yaml:1024]: `<member:id>` ?

File: parameters/member_uuid_path.yml [hunk 5]
```diff
@@ -1,4 +1,4 @@
-name: member_uuid
+name: member_id
 in: path
 description: member_id of the member
 required: true
```
Comment (dazza-codes) [inline parameters/member_uuid_path.yml:4]: rename this file to `parameters/member_id_path.yml` ?

File: paths/index.yml [hunk 6]
```diff
@@ -36,14 +36,14 @@
     $ref: member/get.yml
   post:
     $ref: member/post.yml
-/member/{uuid}:
+/member/{member_id}:
```
Comment (dazza-codes) [inline paths/index.yml:39]: `/member/{id}` ?

**Comment by malakai97** [comment]
`id` fields of ActiveRecord objects are controlled by ActiveRecord, and are automatically allocated, usually by the underlying adapter.  We would not be able to enforce them being the same across nodes.  There are some ways around this, but the solution would call for knowing exactly when a dpn:id is needed and an AR:id is needed.  Too much complexity for the benefit.

**Comment by dazza-codes** [comment]
Can everything use `uuid` instead of `id`?  This PR proposes to bring `member_id` into conformity with other paths.  Coming to this a bit later, I'm curious about why other paths (all paths) do not use a `uuid` as the primary DPN-key?  (I'm aware of one problem with `replication_id`, but that could be replaced by `uuid` when `bag` is used for the bag ID in a replication request).

**Pull request merged**